### PR TITLE
Feature home page

### PR DESCRIPTION
**Pull request by avoevodin**
#1

### CODE REVIEW

**Review by iliadmitriev — CHANGES_REQUESTED** [review]
didn't finished
* requirements.txt 
* translation ru
* rich admin interface for for model Employee (maybe next time)
* verbose name of application employees
* fix comments

File: brigadier/employees/forms.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from django import forms
+from django.utils.translation import gettext_lazy as _
+
+from .models import Employee
+
+
+class EmployeeModelForm(forms.ModelForm):
+    """Model form for creating and updating data of Employee objects.
```
[the remark this answers is not in the document]
Comment (avoevodin) [reply brigadier/employees/forms.py:8]: Fixed

File: brigadier/employees/views.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+from django.views import generic
+from django.urls import reverse
```
Comment (iliadmitriev) [inline brigadier/employees/views.py:2]: should use reverse_lazy function when setting Class field success_url

File: brigadier/employees/views.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+class EmployeeCreateView(generic.CreateView):
+    """View is responsible for displaying a form that helps to
+    create new employees.
+
+    """
+    template_name = 'employee_form.html'
+    model = Employee
+    form_class = EmployeeModelForm
+    success_url = reverse('employees:list')
```
Comment (iliadmitriev) [inline brigadier/employees/views.py:34]: reverse_lazy

File: brigadier/employees/views.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+class EmployeeEditView(generic.UpdateView):
+    """View is responsible for displaying a form
+    for editing exciting employees.
+
+    """
+    template_name = 'employee_form.html'
+    model = Employee
+    form_class = EmployeeModelForm
+    success_url = reverse('employees:list')
```
Comment (iliadmitriev) [inline brigadier/employees/views.py:45]: reverse_lazy

File: brigadier/employees/views.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+
+class EmployeeDeleteView(generic.DeleteView):
+    """View for the form of deleting employees.
+
+    """
+    template_name = 'employee_confirm_delete.html'
+    model = Employee
+    success_url = reverse('employees:list')
```
Comment (iliadmitriev) [inline brigadier/employees/views.py:54]: reverse_lazy

**Review by iliadmitriev — CHANGES_REQUESTED** [review]

**Pull request merged**